sympy/parsing/C/parser.py

```diff
@@ -0,0 +1,239 @@
+from __future__ import unicode_literals, print_function
+from sympy.parsing.sympy_parser import (parse_expr, standard_transformations, implicit_multiplication_application)
+from clang.cindex import (
+    CursorKind,
+    Index,
+    StorageClass,
+    TranslationUnit,
+    TypeKind,
+    UnaryOperator
+)
+
+import argparse
+import os
+import re
+import sys
+
+
+class Writer(object):
+    def __init__(self, out,):
```

Comment:
I think `,` shouldn't be there after `out`, right?

Reply:
Oh I left a comma 
I'll fix that